resources/views/admin/include/content.blade.php

```diff
@@ -0,0 +1,52 @@
+@extends('admin.layout.app')
```

Comment:
- Add content of each page in "pages" folder i.e: "\resources\views\admin\pages\dashboard\index.blade.php"

- Replace dashboard content with 4 cards and remove all below static content. You can use same 4 cards as of figma for labeling and counts. ( https://prnt.sc/NYPDcqkkGWXJ )

- Add another layout which is guest layout for the login screen. Present in shared reference yesterday.